aspnetcore/mvc/views/tag-helpers/built-in/ImageTagHelper.md

```diff
@@ -1,4 +1,18 @@
-﻿# ImageTagHelper
+﻿title: Anchor Tag Helper | Microsoft Docs
+author: Peter Kellner
+description: Shows how to work with Image Tag Helper
```

Comment:
@Rick-Anderson put in this description line so leaving as is for now. It's the same in all three tag helpers.